net/mwan3/files/lib/mwan3/common.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+get_uptime() {
+	local uptime=$(cat /proc/uptime)
```

Comment:
Just did a quick test on 17.01, call to `cat` is not optimised and end up being a fork + execve of /bin/cat
I was going to say use `$(</proc/uptime)`, but it's not working (busybox bug??), so maybe `read uptime < /proc/uptime` (but not sure)

Reply:
ping @feckert 
https://git.busybox.net/busybox/tree/shell/ash.c#n13473
-d was only introduced in august / https://git.busybox.net/busybox/commit/?id=3bef5d89b0c667e9fb7d1d9b44ba9b30d4d084e4
so you can use `read uptime < /proc/uptime`

I can merge it as is if you want and you can fix it later